### PR TITLE
Remove old Ward and refactor to new Ward + Patient-Ward connection on reset

### PR DESCRIPTION
**Pull request by daytona65**
- Remove old Ward and refactor to new Ward
- fixData() method in addressbook to ensure data is consistent and link patients to respective wards.

todo:
- Ward CRUD commands
- Better OOP for AddressBook.fixData()

### CODE REVIEW

File: src/main/java/seedu/address/model/AddressBook.java [hunk 1]
```diff
@@ -68,9 +68,38 @@ public void setWards(List<Ward> wards) {
...
+            for (Ward ward:wards) {
+                if (wardName == ward.value) {
+                    wardExists = true;
+                    if (!ward.hasPatient(patient)) {
+                        ward.addPatient(patient);
+                    }
+                    break;
+                } else if (ward.hasPatient(patient)) {
+                    ward.removePatient(patient);
+                    break;
```
Comment (daytona65) [inline src/main/java/seedu/address/model/AddressBook.java:94]: Some pretty bad nesting here but ill fix it next PR

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 2]
```diff
@@ -6,27 +6,42 @@
...
 /**
  * Contains utility methods for populating {@code AddressBook} with sample data.
  */
 public class SampleDataUtil {
     public static Patient[] getSamplePatients() {
         return new Patient[] {
-            new Patient(new Nric("S1234567A"), new Name("Alex Yeoh")),
-            new Patient(new Nric("S0000000A"), new Name("Bernice Yu")),
+            new Patient(new Nric("S1234567A"), new Name("Alex Yeoh"), new Status("RED"), "Intensive Care"),
+            new Patient(new Nric("S0000000A"), new Name("Bernice Yu"), "Class C"),
```
Comment (daytona65) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:18]: Changed up the sample patients abit

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 3]
```diff
@@ -6,27 +6,42 @@
...
+    public static Ward[] getSampleWards() {
+        return new Ward[] {
+            new Ward("Waiting Room"),
+            new Ward("Class A"),
+            new Ward("Class B"),
+            new Ward("Class C"),
+            new Ward("Intensive Care")
+        };
+    }
+
```
Comment (daytona65) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:35]: Added sample wards

File: src/main/java/seedu/address/model/AddressBook.java (hunk 1 above)
Comment (nramapurath) [reply src/main/java/seedu/address/model/AddressBook.java:94]: Alright!

**Comment by yitong241** [comment]
Nesting has some issue but will merge first

**Pull request merged**